judgels-backends/jophiel/jophiel-api/src/main/java/judgels/jophiel/api/user/UserService.java

```diff
@@ -29,6 +29,11 @@
     @Produces(APPLICATION_JSON)
     User getUser(@HeaderParam(AUTHORIZATION) AuthHeader authHeader, @PathParam("userJid") String userJid);
 
+    @POST
+    @Path("/{userJid}/logout")
```

Comment:
Can we put this in `SessionService` instead?

Something like:

```
POST /api/v2/session/batch-logout
{
   usernames: ["andi", "budi"]
}
```

It is better to have bulk endpoint in the first place.

Reply:
Changed endpoint to be `session/batch-logout` instead in `SessionService`, but with `userJids` parameter instead of `usernames`.